lib/validate.js

```diff
@@ -392,6 +438,16 @@ function validateCreateContainer(req, res, next) {
             'HostConfig.PortBindings');
         assertOptionalObject(container.HostConfig.LogConfig,
             'HostConfig.LogConfig');
+        assertOptionalString(container.HostConfig.NetworkMode,
+            'HostConfig.NetworkMode');
+
+        if (container.HostConfig.NetworkMode) {
+            assertNetworkMode(container.HostConfig.NetworkMode,
+                'HostConfig.NetworkMode');
+        }
+
+        // ensures mutually-compatible networking params.
+        assertNetworkingParams(container.HostConfig);
```

Comment:
If I understand correctly, `assertNetworkParams` returns an error object when networking params are invalid, but this call-site doesn't check for the return value.